Project.toml

```diff
@@ -9,6 +9,7 @@ Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 GaussianProcesses = "891a1506-143c-57d2-908e-e1f8e92e6de9"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
Do we want to keep this dependency on Plots or do something else? @jakebolewski @bielim @odunbar

Reply:
I think a good thing to do with plots (for this PR) is described here https://discourse.julialang.org/t/add-package-to-project-toml-when-needed-only-for-testing/20552/3 
In our setting I think it amounts to the following

1. Add `Test` to the Project.toml
2. Add (manually) new sections `[extras]` and `[targets]`  into Project.toml,
3. Copy the `Test` line and the `Plots` lines into `[extras]`
4. in the `[targets]` section put `test =["Test","Plots"]`

Then when you call Pkg.test("MyPackage"), (not run runtest.jl directly) it will also load the Plots package.

NB I haven't done this before but hopefully it will work

Reply:
Let's try this, yes (this is what worked for @glwagner in Oceananigans, right?)